.circleci/config.yml

```diff
@@ -568,6 +574,23 @@ jobs:
             bundle exec fastlane android create_snapshots_repo_pr
       - android/save-build-cache
 
+  run-revenuecatui-ui-tests:
+    description: "Run RevenueCatUI UI tests for Android in CircleCI"
+    <<: *android-machine-emulator
+    steps:
+      - checkout
+      - install-sdkman
+      - android/create-avd:
+          avd-name: test-revenuecat-ui
+          system-image: system-images;android-32;google_apis;x86_64
+          install: true
+      - android/start-emulator:
+          avd-name: test-revenuecat-ui
+          post-emulator-launch-assemble-command: ./gradlew ui:revenuecatui:assembleDebugAndroidTest
+      - run:
+          command: |
+            ./gradlew ui:revenuecatui:connectedDebugAndroidTest
```

Comment:
Running the predefined CircleCI job to run all instrumentation tests didn't work since it only run integration tests on modules that didn't have flavors. This basically removed most of the instrumentation tests.

In the end I switched to just run the `ui:revenuecatui` module instrumentation tests.